ssg/build_cpe.py

```diff
@@ -84,19 +84,17 @@ def add_cpe_item(self, cpe_item):
     def _is_name(self, ref):
         return ref.startswith("cpe:")
 
-    def _is_parametrized(self, ref):
-        return re.search(r'^\w+\[\w+\]$', ref)
```

Comment:
By applying this change, the re module is no longer needed and does not need to be imported.